packages/code-du-travail-modeles/src/utils/GetReferences.ts

```diff
@@ -0,0 +1,25 @@
+import Engine from "publicodes";
+
+export type References = {
+  article: string;
+  url: string;
+};
+
+export function getReferences(engine: Engine): Array<References> {
+  return Object.values(engine.getParsedRules())
+    .filter(
+      (rule) =>
+        rule.rawNode.références && !!engine.evaluate(rule.dottedName).nodeValue
```

Comment:
not very comfortable with the double `!!`  
what are the possible value of nodeValue when it is truthy  or falsy

Reply:
The nodeValue contains: `number | boolean | string | Record<string, unknown>`

Here we can use `nodeValue !== false` in place of `!!`.

Reply:
I change to 
```js
rule.rawNode.références && engine.evaluate(rule.dottedName).nodeValue !== null
```
and all tests passed :+1:

Reply:
thx